x-pack/plugins/infra/public/components/infrastructure_node_metrics_tables/pod/pod_metrics_table.tsx

```diff
@@ -65,38 +58,50 @@ export const PodMetricsTable = (props: PodMetricsTableProps) => {
     setCurrentPageIndex(0);
   };
 
-  if (isLoading) {
+  if (data.state === 'error') {
     return (
```

Comment:
nit: we don't necessarily need an empty tag here, but it doesn't hurt either
```suggestion
    return data.errors.map((error) => <MetricsTableErrorContent error={error} />);
```

Reply:
But `Element[]` is not a valid return type for components, isn't it?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6ba14506a1d2ceac41d9827a1d194708a2c46a00/types/react/index.d.ts#L520

Reply:
🤔 If we had used `React.FC` here, then the return type would've been enforced. But here we haven't defined the return type, so it wouldn't necessarily complain about it being an `Element[]`

Now thinking more about this, better to leave as it. If we change it and don't specify a return type for the function, the component will return `any`